### DISCUSSION TITLE
What about a Pixi V8 beta forum?

### DISCUSSION THREAD

**Discussion by joergplewe** [pixijs/pixijs#9500] [v8 General]
Dear admired developers,

what about forum to discuss latest development concerning v8?

Would allow us consumers to specifically ask about coming concepts and solutions and Pixi developers to see what is needed and where people struggle.

Personally I'd like to follow v8 development as close as possible helping me migrate when the day comes.

**Comment by joergplewe** [comment]
e.g. things I'd asked myself when trying `v8.0.0-alpha.0`:

* what happened to `ParticleContainer`?
* how is `Text` meant to work concerning texture?
  * as I have thousends of `Sprite`s with a `Texture` taken from a `Text`

**Reply by GoodBoyDigital** [reply]
hey @joergplewe , this is as good a place as any! Im excited to know that someone else is testing it out!!

In answer to your questions:

- `ParticleContainer` is gone, in almost all cases, its slower than the regular rendering! Unless you have like 100000+ particles.
- Textures are now cached under the hood for text, so making 1000 text objects with matching style and text will always use the same texture.
- If you want to just grab a Text Texture to use for a Sprite you can use the new CanvasTextSystem directly:

```
const helloTexture = renderer.canvasText.getTexture('hello', 1, style);

const sprite = new Sprite(helloTexture);
```

any other questions, give me a shout!

**Comment by GoodBoyDigital** [comment]
Oh, also we are trying to keep all changes [here](https://github.com/pixijs/pixijs/releases/tag/v8.0.0-alpha.0) for now
